lib/ui/painting/picture_recorder.cc

```diff
@@ -47,12 +47,14 @@ SkCanvas* PictureRecorder::BeginRecording(SkRect bounds) {
   return picture_recorder_.beginRecording(bounds, &rtree_factory_);
 }
 
-fml::RefPtr<Picture> PictureRecorder::endRecording() {
+fml::RefPtr<Picture> PictureRecorder::endRecording(Dart_Handle dart_picture) {
   if (!isRecording())
     return nullptr;
 
-  fml::RefPtr<Picture> picture = Picture::Create(UIDartState::CreateGPUObject(
-      picture_recorder_.finishRecordingAsPicture()));
+  fml::RefPtr<Picture> picture =
+      Picture::Create(std::move(dart_picture),
```

Comment:
My understanding of `std::move` is quite limited, but is this necessary here and elsewhere? This is simply a `Dart_Handle` which is just a pointer.

Reply:
It's probably not necessary here then - I default to using move whenever possible at this point.

@chinmaygarde wdyt?

Reply:
Here and elsewhere, it is redundant. `clang-tidy` will warn about this. I'd prefer not adding this unnecessarily as we already delete copy constructors on objects that we'd have folks rather not copy.

Reply:
Done - I think I got them all.